scipy/optimize/_minpack_py.py

```diff
@@ -864,6 +869,24 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
     suggesting that the optimal values of these parameters are ambiguous and
     that only one of these parameters is needed in the model.
 
+    If the optimal parameters of `f` differ by more than four orders
```

Comment:
One final query, how did you arrive at four orders of magnitude here? Unless there is a concrete reason I'd perhaps prefer to stick to multiple orders of magnitude language used earlier

Reply:
I believe this was my suggestion. I changed it from five to four based on the example. I agree that multiple orders of magnitude would be a better option.

Reply:
Purely by experimentation. It seems to depend on a lot of things, and method=trf seems to be more robust against scale differences than the default method=lm. Generic "multiple orders" would be correct, though up to 4 orders of magnitude difference seems to work fine with any method.

Reply:
I would prefer a more generic phrasing if that's ok as means we are not promising anything that we aren't 100% on

Reply:
Done :)